cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/dataset/MetadataDataset.java

```diff
@@ -893,10 +901,14 @@ private IndexColumn getIndexColumn(String key, SortInfo.SortOrder sortOrder) {
    * @param metadataEntity target id for which metadata needs snapshotting
    */
   private void writeHistory(MetadataEntity metadataEntity) {
+    writeHistory(metadataEntity, System.currentTimeMillis());
```

Comment:
why a separate method just to pass time?

Reply:
This is needed for writing history with migrated time stamp